shared/nav.desktop.js

```diff
@@ -155,6 +156,7 @@ class Nav extends Component<void, Props, void> {
           <MetaNavigator
             tab={loginTab}
             rootComponent={Login} />
+          <GlobalError />
```

Comment:
talked to cecile and its correct for the error to go across the nav as its a global error and not a pane related thing